packaging/CodeCoverage.spec

```diff
@@ -0,0 +1,35 @@
+Name:          CodeCoverage
+Version:       %{version}
+Release:       1%{?dist}
+Summary:       A wrapper for building and running code coverage on NetMon C++ repositories
+Group:         Development/Tools
+License:       (c) Copyright 2003-2016, LogRhythm, Inc.  All rights reserved.
```

Comment:
@KjellKod is right about this being on a public github account. So we either need to move this to our lrgit repository or change this back to MIT. For now, I'd change it back to MIT.